internal/image/fetcher.go

```diff
@@ -7,9 +7,10 @@ import (
 	"io"
 	"strings"
 
+	"github.com/buildpacks/imgutil/remote"
+
 	"github.com/buildpacks/imgutil"
 	"github.com/buildpacks/imgutil/local"
-	"github.com/buildpacks/imgutil/remote"
 	"github.com/buildpacks/lifecycle/auth"
```

Comment:
As Javier mentioned https://github.com/buildpacks/pack/pull/1187#discussion_r634624030, there should be 3 import groups, I think.